README.md

```diff
@@ -54,10 +54,10 @@ In case following error occurs, it means there is a test module `XYZ` that canno
 
 ```txt
 File "setup.py", line 499, in loadTestsFromModule
-    testsuites = ScanningLoader.loadTestsFromModule(self, module)
+testsuites = ScanningLoader.loadTestsFromModule(self, module)
 File "build/bdist.linux-x86_64/egg/setuptools/command/test.py", line 37, in loadTestsFromModule
 File "/usr/lib64/python2.7/unittest/loader.py", line 100, in loadTestsFromName
-    parent, obj = obj, getattr(obj, part)
+parent, obj = obj, getattr(obj, part)
```

Comment:
The purpose of the original pr that I am fixing here ( #28 ) was to check all README files to render as proper rst

that's all I'm doing here.
I guess we should decide if it's the way we want to go.
currently this file now renders ok as markdown, and doesn't fail on rst,

but if we want to go ahead and merge this, all readme's must become rst.